google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/CloudToolsFeedbackActionTest.java

```diff
@@ -59,23 +56,22 @@ public void testDisplayableSdkVersion_isEmpty_whenSdkIsInvalid() {
     when(sdkService.getSdkHomePath()).thenReturn(missingCloudSdk.toPath());
 
     feedbackAction.actionPerformed(null /*event*/);
-    verify(browserLauncher).browse(urlArg.capture(), eq(null));
 
-    String expectedEmptySdkVersionMessage = "- Google Cloud SDK version: \n";
-    assertTrue(urlContainsMessage(expectedEmptySdkVersionMessage));
+    String expected = "- Google Cloud SDK version: \n";
+    verify(browserLauncher).browse(urlContains(expected), eq(null), eq(null));
```

Comment:
Discussed offline; there's something strange going on here before/after the EAP update and with/without Mockito 2. I'll take a look at this again once the Mockito 2 PR is merged to eliminate variables when debugging.